### PR TITLE
API Blueprint Documentation for Stripe Account

### PR DESCRIPTION
**Pull request by mackenziehicks**
# What's in this PR?

Added Endpoint Stripe Account with index(get) and show(get) and added Data Structure
## References

Fixes: #390 

Progress on: #245

### CODE REVIEW

File: blueprint/api.apib [hunk 1]
```diff
@@ -1227,6 +1229,64 @@ This endpoint will return one of two possible responses, depending on if the slu
...
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Parameters
+
+    + id (number) - Id of a stripe card.
```
Comment (amyschools) [inline blueprint/api.apib]: Capital Stripe

File: blueprint/api.apib [hunk 2]
```diff
@@ -1227,6 +1229,64 @@ This endpoint will return one of two possible responses, depending on if the slu
 
     + Attributes (Slugged Route Organization Response)
 
+# Group Stripe Accounts
+
+This is resource represents a Stripe account. It can be retrieved to see properties on the account like its current e-mail address or if the account is enabled yet to make live charges.
```
Comment (amyschools) [inline blueprint/api.apib]: Wonky grammar?

File: blueprint/api.apib [hunk 3]
```diff
@@ -220,6 +220,8 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Unprocessable Entity Response)
 
+
+
```
Comment (joshsmith) [inline blueprint/api.apib]: Extra spacing.

File: blueprint/api.apib [hunk 4]
```diff
@@ -1227,6 +1229,64 @@ This endpoint will return one of two possible responses, depending on if the slu
 
     + Attributes (Slugged Route Organization Response)
 
+# Group Stripe Accounts
+
+This resource represents a Stripe account. It can be retrieved to see properties on the account like its current e-mail address or if the account is enabled yet to make live charges.
```
Comment (joshsmith) [inline blueprint/api.apib:1232]: Would link to Stripe's API here.

Comment (joshsmith) [reply blueprint/api.apib:1232]: I would also specify the relationship between an account and an organization, how that works.

File: blueprint/api.apib [hunk 5]
```diff
@@ -2485,6 +2545,43 @@ This endpoint allows you to check whether a username is valid (by running a vali
...
+    + organization
+        + data(Organization Resource Identifier)
+
+## Stripe Account Response
++ data(Stripe Account Resource)
++ include JSON API Version
+
+## Stripe Accounts Response
++ data(array[Stripe Account Resource], required)
++ include JSON API Version
```
Comment (joshsmith) [inline blueprint/api.apib:2651]: The `required` here is probably not necessary.

**Review by joshsmith — CHANGES_REQUESTED** [review]
Just minor things I'd change here:

File: blueprint/api.apib [hunk 6]
```diff
@@ -1233,6 +1231,11 @@ This endpoint will return one of two possible responses, depending on if the slu
 
 This resource represents a Stripe account. It can be retrieved to see properties on the account like its current e-mail address or if the account is enabled yet to make live charges.
 
+A Stripe account belongs to an organization.
+
+See
+https://stripe.com/docs/api#account
```
Comment (joshsmith) [inline blueprint/api.apib]: You can use markdown links here. I would keep it similar to how we're writing this in other groups.

**Review by joshsmith — APPROVED** [review]

**Comment by joshsmith** [comment]
👍  just squash and rebase!

**Pull request merged**